### PR TITLE
Updates to book

### PR DESCRIPTION
**Pull request by Emilgardis**
Some changes to the book. 
Updated the way of getting the git version since the way that was described is unnecessary if the user doesn't want to make changes to amethyst.

Also note:

    amethyst = {git = "https://github.com/amethyst/amethyst.git", branch = "develop" } 

This will make cargo use the develop branch.

Changed method of switching between toolchains to rustup.

### CODE REVIEW

**Comment by Emilgardis** [comment]
Also I noted that on the first page there are references like . 

> I've wanted to play around [...]
> Most of those I did find stuck [...]
> In short, I am writing Amethyst to scratch [...]

These should probably be taken out as this is not a single-man project anymore. However, it could be nice to somehow keep the _charm_ it gives to readers.

File: book/src/getting_started.md [hunk 1]
```diff
@@ -19,8 +19,11 @@ Rust compiler. Here are the system requirements (they're pretty modest):
 ## Setting Up
 
 > Note: This guide assumes you have nightly Rust and Cargo installed, and also
-> have a working Internet connection. Please take care of these prerequisites
-> first before proceeding.
+> have a working Internet connection. 
+> Please take care of these prerequisites first before proceeding.
+> See [rustup][ru] for an easy way to have two installations of rust.
```
Comment (Emilgardis) [inline book/src/getting_started.md]: Now when I read this out loud, it doesn't sound that good, any suggestions for an edit?

**Comment by LucioFranco** [comment]
👍 this seems like well-needed update to the book.

**Comment by Emilgardis** [comment]
Took the liberty of amending and then force pushing the previous commit, I had missed one link.

**Comment by LucioFranco** [comment]
LGTM @ebkalderon what do you think?

**Comment by ebkalderon** [comment]
Looks good to me! Thanks for the updates, @Emilgardis. I think we can merge this.

**Pull request merged**